### PR TITLE
Migrate to Java 21

### PR DESCRIPTION
**Pull request by ravinperera00**
## Purpose

Fixes:

## Checklist
- [ ] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Checked native-image compatibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ This Package bundles the latest MSSQL driver so that the mssql connector can be
 
 ### Set up the prerequisites
 
-1. Download and install Java SE Development Kit (JDK) version 17 (from one of the following locations).
-   * [Oracle](https://www.oracle.com/java/technologies/downloads/#java17)
+1. Download and install Java SE Development Kit (JDK) version 32 (from one of the following locations).
```
Comment (warunalakshitha) [inline README.md:13]: 21?

File: ballerina/target/cache/tests_cache/test_suit.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (warunalakshitha) [inline ballerina/target/cache/tests_cache/test_suit.json:1]: no need this file

Comment (ravinperera00) [reply ballerina/target/cache/tests_cache/test_suit.json]: Fixed with 8e43242a17ba9c0ef7c467d471137764fd2b3922

**Review by warunalakshitha — APPROVED** [review]
LGTM

**Pull request merged**